% https://www.overleaf.com/latex/templates/template-for-data-descriptor-submissions-to-scientific-data/qrrnhwspvgbf
% https://www.nature.com/sdata/publish/submission-guidelines
\documentclass[english]{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{babel}
\usepackage{amsmath}
\usepackage{graphicx}
\usepackage{fancyhdr}
\newcommand{\scidatalogo}{\includegraphics[height=36pt]{SciData_logo.jpg}}
\newcommand{\overleaflogo}{\includegraphics[height=36pt]{Overleaf-logo-300dpi.png}}
\pagestyle{fancy}
\fancyhf{}
\renewcommand{\headrulewidth}{0pt}
\setlength{\headheight}{40pt} 
\lhead{\textsc{\scidatalogo}}
\rhead{\textsc{\overleaflogo}}

\usepackage[colorlinks=true, allcolors=blue]{hyperref} % package added -MMM
\usepackage{longtable} % package added -MMM

\begin{document}

\urlstyle{same}
\urlstyle{rm}

%\title{Daily PM\textsubscript{2.5} concentration estimates by ZIP code in 11 western states wildfire and non-wildfire, 2008-2018} 
%\title{Daily wildfire and non-wildfire PM\textsubscript{2.5} concentration estimates by ZIP code in western US states, 2008-2018} 
\title{Machine learning-derived daily wildfire and non-wildfire PM\textsubscript{2.5} concentration estimates over the western US, 2008-2018}
%\title{Data Descriptor Title (110 character maximum, inc. spaces)}

\author{Colleen E. Reid\textsuperscript{1{*}},
Ellen M. Considine\textsuperscript{2},
Melissa M Maestas\textsuperscript{2}, 
Gina Li\textsuperscript{1,2}}
%\author{Firstname Lastname\textsuperscript{1}, Firstname
%Lastname\textsuperscript{2{*}}}

\maketitle
\thispagestyle{fancy}

%1. An affiliation 2. A different affiliation {*}corresponding author(s):
%Firstname Lastname (email@address)
1. Department of Geography, University of Colorado Boulder,
Boulder, Colorado, USA and
2. Cooperative Institute for Research in
Environmental Sciences, Earth Lab %and %, University of Colorado Boulder,
%Boulder, Colorado, USA 
{*}corresponding author: Colleen Reid (Colleen.Reid@Colorado.edu)
%{*}corresponding author(s): Colleen Reid (Colleen.Reid@Colorado.edu)

\begin{abstract}
%TEMPLATE TEXT: This is a manuscript template for Data Descriptor submissions to \emph{Scientific Data} (http://www.nature.com/scientificdata). The abstract must be no longer than 170 words, and should succinctly describe the study, the assay(s) performed, the resulting data, and the reuse potential, but should not make any claims regarding new scientific findings. No references are allowed in this section. 

Fine particulate matter (PM\textsubscript{2.5}) levels are declining in many areas of the US due to policies and enforcement of the Clean Air Act. However, in much of the western US, PM\textsubscript{2.5} concentrations have been increasing, likely due to the increased presence of wildfires in this region. There is growing evidence of various health impacts of PM\textsubscript{2.5} exposures, even at levels below the federal standard. Health studies of PM\textsubscript{2.5} in the western US are limited by spatial sparseness of monitoring data. To improve population exposure assessment of PM\textsubscript{2.5}, researchers are increasingly using statistical methods to  “blend” information from multiple data sources to better estimate PM\textsubscript{2.5} in space and time. Some studies have created daily fine-resolution estimates of PM\textsubscript{2.5} for the whole US, but they perform poorly in the western US. We have tailored a machine learning model to the western US, combining  satellite, meteorological, monitoring, land use and other spatiotemporal data to estimate daily PM\textsubscript{2.5} estimates at the census tract, ZIP code, and county levels during 2008-2018. Our methods improve upon previous models by: use of a more extensive monitoring station network, which captures more spatial locations and proximity to wildfires; use of ensembles of machine learning algorithms, which have been shown to improve model performance; and coverage of a longer period of time. We are making our data publicly available for use in future studies of the health impacts of fine particulate air pollution in the western US.

\end{abstract}

\section*{Background \& Summary}

%TEMPLATE TEXT: (700 words maximum) An overview of the study design, the assay(s) performed, and the created data, including any background information needed to put this study in the context of previous work and the literature. The section should also briefly outline the broader goals that motivated the creation of this dataset and the potential reuse value. We also encourage authors to include a figure that provides a schematic overview of the study and assay(s) design. This section and the other main body sections of the manuscript should include citations to the literature as needed \cite{cite1, cite2}. References should be included within the manuscript file itself as our system cannot accept BibTeX bibliography files. Authors who wish to use BibTeX to prepare their references should therefore copy the reference list from the .bbl file that BibTeX generates and paste it into the main manuscript .tex file (and delete the associated \textbackslash{}bibliography and \textbackslash{}bibliographystyle commands).

Fine particulate matter (PM\textsubscript{2.5}) air pollution is increasingly associated with numerous adverse health outcomes including, but not limited to, mortality \cite{achilleos_acute_2017}, respiratory and cardiovascular morbidity \cite{xing_impact_2016, rajagopalan_air_2018}, negative birth outcomes \cite{klepac_ambient_2018}, and lung cancer \cite{hamra_outdoor_2014}. Although PM\textsubscript{2.5} concentrations have been declining in many parts of the United States due to policies to limit emissions of air pollutants \cite{fann_estimated_2017}, PM\textsubscript{2.5} levels have been increasing in parts of the northwestern US \cite{mcclure_US_2018}. This increase has been shown to be associated with wildfire smoke \cite{mcclure_US_2018, ODell_contribution_2019}, which can cause PM\textsubscript{2.5} concentrations that are several times higher than the Environmental Protection Agency's (EPA’s) daily PM\textsubscript{2.5} National Ambient Air Quality Standard (NAAQS) in areas downwind of the wildfires for several days at a time \cite{reid_associations_2019}. 

Estimates of PM\textsubscript{2.5} concentrations for health studies have traditionally been derived from data from stationary air quality monitors placed in and around populated areas for regulatory purposes. In the US, the EPA’s Federal Reference Method (FRM) monitors often only measure every third or sixth day and do not provide enough spatial coverage to obtain a good estimate of the air pollution exposures where every person lives. In fact, most US counties do not contain a regulatory air pollution monitor \cite{brokamp_assessing_2019}. Using solely monitoring data in health studies leads to exposure misclassification, which often, but not always, drives effect estimates of the association between air pollution and health towards the null \cite{zeger_exposure_2000}.

To improve population exposure assessment of PM\textsubscript{2.5}, epidemiological researchers have increasingly been using methods to estimate PM\textsubscript{2.5} exposures in the temporal and spatial gaps between regulatory monitors using %a 
data from satellites (such as aerosol optical depth (AOD) or polygons of smoke plumes) or air pollution models \cite{brokamp_assessing_2019, LIU2015120}) over the past two decades. Each of these data sources has its own benefits and limitations, and researchers are increasingly statistically “blending” information from a combination of data sources to better estimate PM\textsubscript{2.5} in space and time. Various methods of blending have been used including spatiotemporal regression kriging (e.g., \cite{hu_satellite-based_2019}), geographically-weighted regression (e.g., \cite{lassman_spatial_2017}), and machine learning methods (e.g., \cite{reid_spatiotemporal_2015, hu_estimating_2017, di_assessing_2016}). 

Machine learning methods train large auxiliary datasets, often including satellite %aerosol optical depth (AOD),
AOD, meteorological data, chemical transport model output, and land cover and land use data to provide optimal estimates of PM\textsubscript{2.5} where people breathe. These models have been implemented in various locations around the world at city, regional, and national scales \cite{bellinger_systematic_2017}. Some epidemiological questions can only be addressed in longitudinal studies with large sample sizes. Exposure models with large spatial and temporal domains will help enable such studies. Within the US, Di et al. \cite{di_assessing_2016, di_ensemble-based_2019} and Hu et al. \cite{hu_estimating_2017} have separately used machine learning algorithms to create fine-resolution daily PM\textsubscript{2.5} estimates for the continental US. These models, however, have performed poorly in the western US  \cite{di_assessing_2016,hu_estimating_2017} and particularly the mountain west \cite{di_ensemble-based_2019} compared to the rest of the country. 
Given the increasing trends in PM\textsubscript{2.5} concentrations in parts of the western US and the importance of wildfires as a source of PM\textsubscript{2.5} there, it is important to have a model that is tailored to this region to capture the variability in space and time in this region.

The dataset we describe here improves upon previous daily estimates of PM\textsubscript{2.5} concentrations from machine learning models in the following ways: (1) use of a more extensive monitoring station network than used in previous models that captures more spatial locations and also proximity to wildfires, a key driver of PM\textsubscript{2.5} in the western US, (2) use of an ensemble of machine learning algorithms which have been shown to improve model performance \cite{di_ensemble-based_2019}, (3) better temporal prediction through the use of a nonlinear function (cosine) on day of year, (4) allowance for different prediction models for fire-affected and non-fire affected days to better capture and predict high PM\textsubscript{2.5} levels during wildfires, and (5) incorporation of errors in prediction back into daily estimates through spatial interpolation. We are making these data available as daily estimates of PM\textsubscript{2.5} exposures at census tract, ZIP-code, county scales in a public repository, which the above cited papers have not done, to be used in future studies of the societal impacts of air pollution exposure in the western US, where wildfires are a significant contributor to PM\textsubscript{2.5} concentrations.

[describe Figure \ref{fig:MonitorLocations}:  monitor locations (points) and state boundaries] % Generated using  Map_Monitor_Locations.R

[insert Table 1: list variables]

\section*{Methods}

%TEMPLATE TEXT: The Methods should include detailed text describing any steps or procedures used in producing the data, including full descriptions of the experimental design, data acquisition assays, and any computational processing (e.g. normalization, image feature extraction). See the detailed section in our submission guidelines for advice on writing a transparent and reproducible methods section. Related methods should be grouped under corresponding subheadings where possible, and methods should be described in enough detail to allow other researchers to interpret and repeat, if required, the full study. Specific data outputs should be explicitly referenced via data citation (see Data Records and Citing Data, below). Authors should cite previous descriptions of the methods under use, but ideally the method descriptions should be complete enough for others to understand and reproduce the methods and processing steps without referring to associated publications. There is no limit to the length of the Methods section.

\subsection*{Study Area}

Our study area includes 11 western US states: Arizona, California, Colorado, Idaho, Montana, Nevada, New Mexico, Oregon, Utah, Washington, and Wyoming (Figure \ref{fig:MonitorLocations}). Our temporal domain were all days between January 1, 2008 and ***, 2018. We predicted daily estimates of PM\textsubscript{2.5} at the ZIP code and county levels from machine learning ensembles trained on observed daily PM\textsubscript{2.5} values from monitoring stations from a variety of sources (**put in all PM\textsubscript{2.5} data sources). The predictor variables for the machine learning ensemble included (**put in all variables here) %Ellen. 
More information on the sources of these data can be found in Table \ref{tab:Table1}.  

%Example of citation: \cite{liu_estimating_2005}

\subsection*{PM\textsubscript{2.5} Measurements}
To get a more comprehensive set of locations and time points of PM\textsubscript{2.5} measurement throughout the western US, we did an extensive search for as many PM\textsubscript{2.5} monitoring data within our spatial and temporal study area as we could find. We downloaded PM\textsubscript{2.5} data from the US EPA AQS Air Data Query Tool \cite{EPAAirData2017}  for the 11-state region (Figure \ref{fig:MonitorLocations}) including any of the following parameter codes: 88101, 88500, 88502, 81104 \cite{EPANPM25Memo2017,EPANPM25Parameters2017,EPANAllParameters2017}. These data include the IMPROVE monitors that capture air quality information in more rural areas \cite{EPANPM25IMPROVE2017}. We also retrieved all available PM\textsubscript{2.5} data in the Fire Cache Smoke Monitor Archive (\url{https://wrcc.dri.edu/cgi-bin/smoke.pl}), which includes U.S. Forest Service monitors that were deployed to capture air quality impacts during wildfire events. 

Some states have additional PM\textsubscript{2.5} monitors beyond those required by the U.S. EPA. We reached out to the department charged with air quality in every state within our study domain and obtained additional PM\textsubscript{2.5} data from California Air Resources Board and the Utah Department of Environmental Quality. We only included data that was in addition to the monitors in those states that was part of the U.S. EPA's AQS and IMPROVE data.   

We also reached out to researchers who may have had their own monitoring networks of PM\textsubscript{2.5} throughout the region. We were able to obtain data from the Uintah Basin, Utah from Seth Lyman at Utah State University, and PM\textsubscript{2.5} measurements from the Persistent Cold Air Pool Study (PCAPS) \cite{Silcox_wintertime_2012} conducted in the Salt Lake Valley, Utah in January--February, 2011 from Dr. Geoff Silcox at the University of Utah.  

All of this yielded a total of XX daily PM\textsubscript{2.5} observations, which represent XX locations. %Ellen

\subsection*{Predictor Variables}

[Write short description of each predictor data set and refer to Table 1]

Satellite Aerosol Optical Depth (AOD) is a measure of particle loading in the atmosphere from the ground to the satellite, which can be used as a proxy measure of PM\textsubscript{2.5} when combined with other variables such as ... We obtained daily estimates of %Aerosol Optical Depth (AOD)from 
AOD from the MODIS Terra and Aqua combined Multi-angle Implementation of Atmospheric Correction (MAIAC) dataset \url{https://ladsweb.modaps.eosdis.nasa.gov/archive/allData/6/MCD19A2/}. This is the finest resolution (1 km) AOD dataset currently available and was available for our whole time period and spatial domain. After downloading each 
Hierarchical Data Format 
(HDF) file from the online repository, we calculated the average daily AOD values at each location, and took the nearest neighbor value at each PM\textsubscript{2.5} monitoring location. MAIAC AOD has been shown to better predict PM\textsubscript{2.5} than coarser resolution AOD \cite{chudnovsky_spatial_2012} and has been used in many studies in various geographic regions in blended models to predict daily PM\textsubscript{2.5} \cite{lee_benefits_2019, geng_satellite-based_2018-1, li_using_2018}.

We obtained meteorological data from the North American Mesoscale (NAM), Analysis meteorological model \url{https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/north-american-mesoscale-forecast-system-nam} because it includes all of the standard meteorological variables, including planetary boundary layer height, which are important for converting AOD to PM\textsubscript{2.5} \cite{liu_estimating_2005}. We calculated 24-hour averages from 6-hourly data for temperature, relative humidity, sea level pressure, surface pressure, planetary boundary layer height, dew point temperature, precipitation, snow coverage, and the U and V components of wind speed. NAM has 12 km resolution and is available from 2004 onward.

Because one of the reasons that PM\textsubscript{2.5} concentrations have been increasing in the western US is the increasing number and magnitude of wildfires, we wanted to have variables about the proximity of a location to an active fire. 
We collected daily data about fire detection locations and size from the MODIS Thermal Anomalies/Fire Daily L3 Global 1km product (MOD14 and MYD14) \cite{Giglio2006,Hawbaker2017}. 
%We collected daily data about fire detection locations, size, and fire radiative power from the MODIS Thermal Anomalies/Fire Daily L3 Global 1km product (MOD14 and MYD14) \cite{Giglio2006,Hawbaker2017}. 
As fires in closer proximity are likely to influence PM\textsubscript{2.5} more than fires further away, we calculated the number of active fires in radial buffers of 25, 50, 100, and 500 km radii around each monitoring location. 
%since we did not use FRP, should we remove this? Is FRP a variable in our model?

Elevation can influence PM\textsubscript{2.5} concentrations. For example, PM\textsubscript{2.5} can accumulate in mountain valleys during persistent cold air pools (commonly referred to as inversions) 
during winter \cite{Whiteman2014}. We obtained elevation data from the 3D Elevation Program, which has a resolution of 1 arc-second, which is approximately 30 m north/south and varies east/west with latitude \cite{USGSElevation2017}.

Surrounding land cover can be a proxy for air pollution emissions. We used the land cover class information from the Landsat-derived National Land Cover Dataset (NLCD) \cite{Homer2017} to calculate the percentage of urban development (codes 22, 23, and 24), agriculture (codes 81 and 82), and vegetated area other than agricultural land (codes 21, 41, 42, 43, 52, and 71) within buffer radii of 1 km, 5 km, and 10 km around each monitor. NLCD 2011 has a spatial resolution of 30 m and uses circa 2011 Landsat satellite data. We obtained the Normalized Difference Vegetation Index (NDVI) from the MODIS satellite product MOD13A3 \url{https://lpdaac.usgs.gov/products/mod13a3v006/} at 1 km resolution by month as another measure of vegetation that was not just a measure of agricultural vegetation but all vegetation. 

%To estimate emissions from vehicles, 
As a proxy indicator of emissions from vehicles, we calculated the sum of all road lengths of type Arterial and Collector within 100, 250, 500, 1000 m buffers of each monitoring location. Arterial roads are high-capacity urban roads. Collector roads are low-to-moderate capacity roads. The road data came from the National Highways Planning Network \url{https://www.fhwa.dot.gov/planning/processes/tools/nhpn/index.cfm} which contains spatial information on over 450,000 miles of highways in the United States. 

We included population density as an additional proxy for emisions as areas with higher population have more sources of air pollution emissions. Population density was obtained from the American Community Survey at the XXX (spatial resolution) for each year or five year averages? 
%Ellen, could you add to this about population density - I don't think that it is in our documentation

To account for seasonality in PM\textsubscript{2.5} data, we created the following predictor variables:  cosine of day-of-year and cosine of month. %Ellen, can you explain why you chose cosine for these?
We also created dummy variables for each state and month in our study domain to allow for spatial and temporal variation in the data that could not be explained by any of the other spatial, temporal, or spatiotemporal variables. 
%Let's move this comment below to the machine learning methods section - done
%Finally, we created an indicator variable for whether there were one or more fires within 500 km of a monitor in the last week. 

\subsection*{Data merging}
We created three datasets: one dataset to train the model and two prediction datasets. The training dataset merged all predictor variables to each 24-hour average PM\textsubscript{2.5} monitoring observation by linking the data temporally (using date) and spatially (by selecting the nearest observation for each predictor variable). Similarly, the prediction datasets were created by spatially and temporally linking all predictor variables to the population-weighted centroid of each ZIP code and county for each day in the study domain. 

\subsection*{Machine learning modeling and mapping}

%Ellen - for now, I am going to leave this as is and let you put this in once we have this set. 

%Let's move this comment below to the machine learning methods section
Finally, we created an indicator variable for whether there were one or more fires within 500 km of a monitor in the last week. 

\subsection*{Code availability}

%TEMPLATE TEXT: For all studies using custom code in the generation or processing of datasets, a statement must be included in the Methods section, under the subheading "Code availability", indicating whether and how the code can be accessed, including any restrictions to access. This section should also include information on the versions of any software used, if relevant, and any specific variables or parameters used to generate, test, or process the current dataset. 

[Insert brief description of how to access code on GitHub.] The code was written and annotated in R [version number] and Python [version number] and is available from GitHub [doi citation link]. The key package for implementing the ML model was [caretEnsemble?]. 

\section*{Data Records}

%TEMPLATE TEXT: The Data Records section should be used to explain each data record associated with this work, including the repository where this information is stored, and to provide an overview of the data files and their formats. Each external data record should be cited numerically in the text of this section, for example \cite{cite3, cite4, cite5, cite6}, and included in the main reference list as described below.. A data citation should also be placed in the subsection of the Methods containing the data-collection or analytical procedure(s) used to derive the corresponding record.

%TEMPLATE TEXT: Tables should be used to support the data records, and should clearly indicate the samples and subjects (study inputs), their provenance, and the experimental manipulations performed on each (please see Tables and Submitting Experimental Metadata, below). They should also specify the data output resulting from each data-collection or analytical step, should these form part of the archived record.

All data are freely available from [repository name, data doi citation]. We provide ... [reference Figure 2]

[insert Figure 2: choropleths at zip code level - 4-panel: a) highest year PM\textsubscript{2.5}, Aug or Sept, b) highest year PM\textsubscript{2.5}, Jan/Feb, c) lowest year PM\textsubscript{2.5}, Aug or Sept, d) lowest year PM\textsubscript{2.5}, Jan/Feb.] % Generated using Choropleths_at_ZipCodes_4panel.R

[insert Figure 3: Time series of select cities] % Generated using TimeSeries_SelectCities.R

[Insert Table 3: list of files]

\section*{Technical Validation}

%TEMPLATE TEXT: This section presents any experiments or analyses that are needed to support the technical quality of the dataset. This section may be supported by up figures and tables, as needed. This is a required section; authors must present information justifying the reliability of their data.

[Write description of goodness of fit methods/metrics - out-of-bag data, RMSE, R\textsuperscript{2}, models run on subsets of data, etc.]

[Insert Figure 4: a) out-of bag observed PM\textsubscript{2.5} vs predicted, b) full model observed PM\textsubscript{2.5} vs predicted, c-j) various subsets of data - oob and full model plots (see figure 5 of example paper)]

[Write discussion about variable importance, possibly referring to the suggested figure of variable importance panel figure. Could make an observation or two about the complexity of the variables, e.g., PM\textsubscript{2.5} can be highest at highest and lowest temperatures (summer fire season and winter inversions), etc.]

[Thoughts - insert figure of predicted PM\textsubscript{2.5} vs predictor variable for the 8 (or so) most important variables (panel figure)]

Thoughts: compare to PM\textsubscript{2.5}. Concerned comparing to HMS will take too long? 


\section*{Usage Notes}

%TEMPLATE TEXT: The Usage Notes should contain brief instructions to assist other researchers with reuse of the data. This may include discussion of software packages that are suitable for analysing the assay data files, suggested downstream processing steps (e.g. normalization, etc.), or tips for integrating or comparing the data records with other datasets. Authors are encouraged to provide code, programs or data-processing workflows if they may help others understand or use the data. Please see our code availability policy for advice on supplying custom code alongside Data Descriptor manuscripts.

%TEMPLATE TEXT: For studies involving privacy or safety controls on public access to the data, this section should describe in detail these controls, including how authors can apply to access the data, what criteria will be used to determine who may access the data, and any limitations on data use. 

[Write brief description of things the provided code can be adapted to do, such as making plots of specific years, use in health/pollution studies.]

\section*{Acknowledgements}

%TEMPLATE TEXT: The Acknowledgements should contain text acknowledging non-author contributors. Acknowledgements should be brief, and should not include thanks to anonymous referees and editors or effusive comments. Grant or contribution numbers may be acknowledged.

[Write acknowledgements text here.]

\section*{Author contributions}

%TEMPLATE TEXT: Each author’s contribution to the work should be described briefly, on a separate line, in the Author Contributions section. 

[Write brief description of contribution from each author.]

\section*{Competing interests}

%TEMPLATE TEXT: A competing interests statement is required for all papers accepted by and published in \emph{Scientific Data}. If there is no conflict of interest, a statement declaring this must still be included in the manuscript.

The authors declare not competing interests.

\section*{Figures and figures legends}

%TEMPLATE TEXT: Figure should be referred to using a consistent numbering scheme through the entire Data Descriptor. For initial submissions, authors may choose to supply this document as a single PDF with embedded figures, but separate figure image files must be provided for revisions and accepted manuscripts. In most cases, a Data Descriptor should not contain more than three figures, but more may be allowed when needed. We discourage the inclusion of figures in the Supplementary Information \textendash{} all key figures should be included here in the main Figure section. 

%TEMPLATE TEXT: Figure legends begin with a brief title sentence for the whole figure and continue with a short description of what is shown in each panel, as well as explaining any symbols used. Legend must total no more than 350 words, and may contain literature references. 

[All figures go here and are referred to in the text]

%%%%% START CODE FOR FIGURE 1 %%%%%%

\begin{figure} 
\centering  
\includegraphics[width=0.77\textwidth]{Monitor_Locations.jpg} 
\caption{\label{fig:MonitorLocations}Locations of PM\textsubscript{2.5} monitoring stations that had at least one observation during our study period (2008-2018).} 
\end{figure} 

%%%%% END CODE FOR FIGURE 1 %%%%%%%


\section*{Tables}

%TEMPLATE TEXT: Authors are encouraged to provide one or more tables that provide basic information on the main ‘inputs’ to the study (e.g. samples, participants, or information sources) and the main data outputs of the study; also see the additional information on providing metadata on page 6. Tables in the manuscript should generally not be used to present primary data (i.e. measurements). Tables containing primary data should be submitted to an appropriate data repository.

%TEMPLATE TEXT: Tables may be provided within the \LaTeX{} document or as separate files (tab-delimited text or Excel files). Legends, where needed, should be included here. Generally, a Data Descriptor should have fewer than ten Tables, but more may be allowed when needed. Tables may be of any size, but only Tables which fit onto a single printed page will be included in the PDF version of the article (up to a maximum of three). 

%TEMPLATE TEXT: Due to typesetting constraints, tables that do not fit onto a single A4 page cannot be included in the PDF version of the article and will be made available in the online version only. Any such tables must be labelled in the text as ‘Online-only’ tables and numbered separately from the main table list e.g. ‘Table 1, Table 2, Online-only Table 1’ etc.

%[All tables go here and are referred to in the text - read template text for tables]

%%%% START TABLE 1 VARIABLE LIST %%%%

\begin{longtable}{l|l|l} \caption{Variables used in the machine learning models.} \label{tab:Table1} \\ 
\hline 
\textbf{Variables}  & \textbf{Type}  & \textbf{Source}  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Coordinates in degrees (Latitude \\and Longitude)\end{tabular}  & Spatial  & \begin{tabular}[c]{@{}l@{}}PM2.5 monitoring \\data\end{tabular}  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Count of Active Fire Points within \\each of the following radial \\buffers (25 km, 50 km, 100 \\km, and 500 km) for same day \\and up to 7 lag days for each \\radial buffer\end{tabular}  & Spatiotemporal  & \begin{tabular}[c]{@{}l@{}}MODIS Thermal Anomalies/Fire \\Daily L3 Global \\1km product\end{tabular}  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Binary Fire indicator (0 for no \\active fire points in any buffer \\radii or lag for given point; \\1 otherwise)\end{tabular}  & Spatiotemporal  & \begin{tabular}[c]{@{}l@{}}MODIS Thermal Anomalies/Fire \\Daily L3 Global \\1km product\end{tabular}  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Summed length (in meters) of arterial \\(A) and collector (C) \\roads and both (A + C) within \\100, 250, 500, and 1000 m buffer \\radii\end{tabular}  & Spatial  & \begin{tabular}[c]{@{}l@{}}National Highways Planning \\Network \end{tabular}  \\ 
 \hline 
Population Density  & Spatial  & \begin{tabular}[c]{@{}l@{}}**Ellen, can you provide \\this information?\end{tabular}  \\ 
 \hline 
AOD (unitless)  & Spatiotemporal  & \begin{tabular}[c]{@{}l@{}}MODIS Terra and Aqua \\combined Multi-angle \\Implementation of \\Atmospheric Correction \\(MAIAC) dataset\end{tabular}  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Planetary Boundary Layer Height \\(m)\end{tabular}  & Spatiotemporal  & NAM  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Temperature at 2 m above ground \\(K)\end{tabular}  & Spatiotemporal  & NAM  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Relative humidity at 2 m above \\ground (\%)\end{tabular}  & Spatiotemporal  & NAM  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Dew point temperature at 2 m above \\ground (K)\end{tabular}  & Spatiotemporal  & NAM  \\ 
 \hline 
Snow Cover (\%)  & Spatiotemporal  & NAM  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}U-component (east/west) of wind \\at 10 m above ground (m/s)\end{tabular}  & Spatiotemporal  & NAM  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}V-component (north/south) of wind \\at 10 m above ground (m/s)\end{tabular}  & Spatiotemporal  & NAM  \\ 
 \hline 
Mean sea level pressure (Pa)  & Spatiotemporal  & NAM  \\ 
 \hline 
Surface pressure (Pa)  & Spatiotemporal  & NAM  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Vertical Wind Velocity (Geometric) \\at 850 mb (m/s)\end{tabular}  & Spatiotemporal  & NAM  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Vertical Wind Velocity (Geometric) \\at 700 mb (m/s)\end{tabular}  & Spatiotemporal  & NAM  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}\% of urban development within \\each of the following radial \\buffers (1 km, 5 km, and 10 \\km)\end{tabular}  & Spatial  & \begin{tabular}[c]{@{}l@{}}National Land Cover \\Database\end{tabular}  \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Normalized Difference Vegetation \\Index (NDVI)\end{tabular}  & Spatiotemporal  & MODIS   \\ 
 \hline 
\begin{tabular}[c]{@{}l@{}}Season (Winter = December-February; \\Spring = March-May; Summer \\= \\June-August; Fall = September-November)\end{tabular}  & Temporal  & derived from date  \\ 
 \hline 
Indicator variables for state  & Spatial  & \begin{tabular}[c]{@{}l@{}}derived from latitude \\and longitude\end{tabular}  \\ 
 \hline 
Indicator variable for year  & Temporal  & derived from date  \\ 
 \hline 
Month  & Temporal  & derived from date  \\ 
 \hline 
Cosine of Day of Week  & Temporal  & derived from date  \\ 
 \hline 
Cosine of Day of Year  & Temporal  & derived from date  \\ 
 \hline 
Elevation  & Spatial  & NED  \\ 
 \hline 
\end{longtable} 

%%%% END OF TABLE 1 VARIABLE LIST %%%%


%%%% top of EXAMPLE BIBLIOGRAPHY - KEEP %%%%
% INSTRUCTIONS: References should be included within the manuscript file itself as our system cannot accept BibTeX bibliography files. Authors who wish to use BibTeX to prepare their references should therefore copy the reference list from the .bbl file that BibTeX generates and paste it into the main manuscript .tex file (and delete the associated \textbackslash{}bibliography and \textbackslash{}bibliographystyle commands).
%\begin{thebibliography}{1}
%\expandafter\ifx\csname url\endcsname\relax
%  \def\url#1{\texttt{#1}}\fi
%\expandafter\ifx\csname urlprefix\endcsname\relax\def\urlprefix{URL }\fi
%\providecommand{\bibinfo}[2]{#2}
%\providecommand{\eprint}[2][]{\url{#2}}
%
%\bibitem{cite1}
%\bibinfo{author}{Califano, A.}, \bibinfo{author}{Butte, A.~J.},
%  \bibinfo{author}{Friend, S.}, \bibinfo{author}{Ideker, T.} \&
%  \bibinfo{author}{Schadt, E.}
%\newblock \bibinfo{title}{{Leveraging models of cell regulation and GWAS data
%  in integrative network-based association studies}}.
%\newblock \emph{\bibinfo{journal}{Nat. Genet.}}
%  \textbf{\bibinfo{volume}{44}}, \bibinfo{pages}{841--847}
%  (\bibinfo{year}{2012}).
%
%\bibitem{cite2}
%\bibinfo{author}{Wang, R.} \emph{et~al.}
%\newblock \bibinfo{title}{{PRIDE Inspector: a tool to visualize and validate MS
%  proteomics data.}}
%\newblock \emph{\bibinfo{journal}{Nat. Biotechnol}}
%  \textbf{\bibinfo{volume}{30}}, \bibinfo{pages}{135--137}
%  (\bibinfo{year}{2012}).
%
%\bibitem{cite3}
%Zhang, Q-L., Chen, J-Y., Lin, L-B., Wang, F., Guo, J., Deng, X-Y. Characterization of ladybird Henosepilachna vigintioctopunctata transcriptomes across various life stages. \emph{figshare} https://doi.org/10.6084/m9.figshare.c.4064768.v3 (2018).
%
%\bibitem{cite4}
%\emph{NCBI Sequence Read Archive} http://identifiers.org/ncbi/insdc.sra:SRP121625 (2017).
%
%\bibitem{cite5}
%Barbosa, P., Usie, A. and Ramos, A. M. Quercus suber isolate HL8, whole genome shotgun sequencing project. \emph{GenBank} http://identifiers.org/ncbi/insdc:PKMF00000000 (2018).
%
%\bibitem{cite6}
%\emph{DNA Data Bank of Japan} http://trace.ddbj.nig.ac.jp/DRASearch/submission?acc=DRA004814 (2016).
%
%\end{thebibliography}
%%%% bottom of EXAMPLE BIBLIOGRAPHY - KEEP %%%%

%%%% Bibliography - copy .bbl file into example above and delete next two lines %%%%
\bibliographystyle{plain}
\bibliography{../ReidGroupReferences}

%\subsection*{Citing Data}
%In line with emerging industry-wide standards for data citation, references to all datasets described or used in the manuscript should be cited in the text with a superscript number and listed in the ‘References’ section in the same manner as a conventional literature reference. See the examples above.

\end{document}
