% https://www.overleaf.com/latex/templates/template-for-data-descriptor-submissions-to-scientific-data/qrrnhwspvgbf
% https://www.nature.com/sdata/publish/submission-guidelines
\documentclass[english]{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{babel}
\usepackage{amsmath}
\usepackage{graphicx}
\usepackage{fancyhdr}
\newcommand{\scidatalogo}{\includegraphics[height=36pt]{SciData_logo.jpg}}
\newcommand{\overleaflogo}{\includegraphics[height=36pt]{Overleaf-logo-300dpi.png}}
\pagestyle{fancy}
\fancyhf{}
\renewcommand{\headrulewidth}{0pt}
\setlength{\headheight}{40pt} 
\lhead{\textsc{\scidatalogo}}
\rhead{\textsc{\overleaflogo}}

\usepackage[colorlinks=true, allcolors=blue]{hyperref} % package added -MMM

\begin{document}

\urlstyle{same}
\urlstyle{rm}

%\title{Daily PM\textsubscript{2.5} concentration estimates by ZIP code in 11 western states wildfire and non-wildfire, 2008-2018} 
%\title{Daily wildfire and non-wildfire PM\textsubscript{2.5} concentration estimates by ZIP code in western US states, 2008-2018} 
\title{Machine learning-derived daily wildfire and non-wildfire PM\textsubscript{2.5} concentration estimates over the western US, 2008-2018}
%\title{Data Descriptor Title (110 character maximum, inc. spaces)}

\author{Melissa M Maestas\textsuperscript{1}, 
Colleen Reid\textsuperscript{2{*}},
Ellen Considine\textsuperscript{1},
Gina Li\textsuperscript{1}}
%\author{Firstname Lastname\textsuperscript{1}, Firstname
%Lastname\textsuperscript{2{*}}}

\maketitle
\thispagestyle{fancy}

%1. An affiliation 2. A different affiliation {*}corresponding author(s):
%Firstname Lastname (email@address)
1. Cooperative Institute for Research in
Environmental Sciences, Earth Lab and %, University of Colorado Boulder,
%Boulder, Colorado, USA 
2. Department of Geography, University of Colorado Boulder,
Boulder, Colorado, USA 
{*}corresponding author(s): Colleen Reid (Colleen.Reid@Colorado.edu)

\begin{abstract}
%TEMPLATE TEXT: This is a manuscript template for Data Descriptor submissions to \emph{Scientific Data} (http://www.nature.com/scientificdata). The abstract must be no longer than 170 words, and should succinctly describe the study, the assay(s) performed, the resulting data, and the reuse potential, but should not make any claims regarding new scientific findings. No references are allowed in this section. 

[Insert abstract text here.]

\end{abstract}

\section*{Background \& Summary}

%TEMPLATE TEXT: (700 words maximum) An overview of the study design, the assay(s) performed, and the created data, including any background information needed to put this study in the context of previous work and the literature. The section should also briefly outline the broader goals that motivated the creation of this dataset and the potential reuse value. We also encourage authors to include a figure that provides a schematic overview of the study and assay(s) design. This section and the other main body sections of the manuscript should include citations to the literature as needed \cite{cite1, cite2}. References should be included within the manuscript file itself as our system cannot accept BibTeX bibliography files. Authors who wish to use BibTeX to prepare their references should therefore copy the reference list from the .bbl file that BibTeX generates and paste it into the main manuscript .tex file (and delete the associated \textbackslash{}bibliography and \textbackslash{}bibliographystyle commands).

Fine particulate matter (PM\textsubscript{2.5}) air pollution is increasingly associated with numerous adverse health outcomes including, but not limited to, mortality \cite{achilleos_acute_2017}, respiratory and cardiovascular morbidity \cite{xing_impact_2016, rajagopalan_air_2018}, negative birth outcomes \cite{klepac_ambient_2018}, and lung cancer \cite{hamra_outdoor_2014}. Although PM\textsubscript{2.5} concentrations have been declining in many parts of the United States due to policies to limit emissions of air pollutants \cite{fann_estimated_2017}, PM\textsubscript{2.5} levels have been increasing in parts of the northwestern US \cite{mcclure_US_2018}. This increase has been shown to be associated with wildfire smoke \cite{mcclure_US_2018, ODell_contribution_2019}, which can cause PM\textsubscript{2.5} concentrations that are several times higher than the Environmental Protection Agency's (EPA’s) daily PM\textsubscript{2.5} National Ambient Air Quality Standard (NAAQS) in areas downwind of the wildfires for several days at a time \cite{reid_associations_2019}. 

Estimates of PM\textsubscript{2.5} concentrations for health studies have traditionally been derived from data from stationary air quality monitors placed in and around populated areas for regulatory purposes. In the US, the EPA’s Federal Reference Monitors (FRMs) often only measure every third or sixth day and do not provide enough spatial coverage to obtain a good estimate of the air pollution exposures where every person lives. In fact, most US counties do not contain a regulatory air pollution monitor \cite{brokamp_assessing_2019}. Using solely monitoring data in health studies leads to exposure misclassification, which often, but not always, drives effect estimates of the association between air pollution and health towards the null \cite{zeger_exposure_2000}.

To improve population exposure assessment of PM\textsubscript{2.5}, epidemiological researchers have increasingly been using methods to estimate PM\textsubscript{2.5} exposures in the temporal and spatial gaps between regulatory monitors using a data from satellites (such as AOD or polygons of smoke plumes) or air pollution models \cite{brokamp_assessing_2019, LIU2015120}) over the past two decades. Each of these data sources has its own benefits and limitations, and researchers are increasingly statistically “blending” information from a combination of data sources to better estimate PM\textsubscript{2.5} in space and time. Various methods of blending have been used including spatiotemporal regression kriging (e.g., \cite{hu_satellite-based_2019}, geographically-weighted regression (e.g., \cite{lassman_spatial_2017}, and machine learning methods (e.g., \cite{reid_spatiotemporal_2015, hu_estimating_2017, di_assessing_2016}). 

Machine learning methods train large auxiliary datasets, often including satellite aerosol optical depth (AOD), meteorological data, chemical transport model output, and land cover and land use data to provide optimal estimates of PM\textsubscript{2.5} where people breathe. These models have been implemented in various locations around the world at city, regional, and national scales \cite{bellinger_systematic_2017}. Some epidemiological questions can only be addressed in longitudinal studies with large sample sizes. Exposure models with large spatial and temporal domains will help enable such studies. Within the US, Di et al. \cite{di_assessing_2016, di_ensemble-based_2019} and Hu et al. \cite{hu_estimating_2017} have separately used machine learning algorithms to create fine-resolution daily PM\textsubscript{2.5} estimates for the continental US. These models, however, have performed poorly in the western US  \cite{di_assessing_2016,hu_estimating_2017} and particularly the mountain west \cite{di_ensemble-based_2019} compared to the rest of the country. Given the increasing trends in PM\textsubscript{2.5} concentrations in parts of the western US and the importance of wildfires as a source of PM\textsubscript{2.5} there, it is important to have a model that is tailored to this region to capture the variability in space and time in this region.

The dataset we describe here improves upon previous daily estimates of PM\textsubscript{2.5} concentrations from machine learning models in the following ways: (1) use of a more extensive monitoring station network than used in previous models that captures more spatial locations and also proximity to wildfires, a key driver of PM\textsubscript{2.5} in the western US, (2) use of an ensemble of machine learning algorithms which have been shown to improve model performance \cite{di_ensemble-based_2019}, (3) better temporal prediction through the use of a nonlinear function (cosine) on day of year, (4) allowance for different prediction models for fire-affected and non-fire affected days to better capture and predict high PM\textsubscript{2.5} levels during wildfires, and (5) incorporation of errors in prediction back into daily estimates through spatial interpolation. We are making these data available as daily estimates of PM\textsubscript{2.5} exposures at census tract, ZIP-code, county scales in a public repository, which the above cited papers have not done, to be used in future studies of the societal impacts of air pollution exposure in the western US, where wildfires are a significant contributor to PM\textsubscript{2.5} concentrations.

[insert Figure 1:  monitor locations (points) and state boundaries]

[insert Table 1: list variables]

\section*{Methods}

%TEMPLATE TEXT: The Methods should include detailed text describing any steps or procedures used in producing the data, including full descriptions of the experimental design, data acquisition assays, and any computational processing (e.g. normalization, image feature extraction). See the detailed section in our submission guidelines for advice on writing a transparent and reproducible methods section. Related methods should be grouped under corresponding subheadings where possible, and methods should be described in enough detail to allow other researchers to interpret and repeat, if required, the full study. Specific data outputs should be explicitly referenced via data citation (see Data Records and Citing Data, below). Authors should cite previous descriptions of the methods under use, but ideally the method descriptions should be complete enough for others to understand and reproduce the methods and processing steps without referring to associated publications. There is no limit to the length of the Methods section.

\subsection*{Study Area}

Our study area includes 11 western US states: Arizona, California, Colorado, Idaho, Montana, Nevada, New Mexico, Oregon, Utah, Washington, and Wyoming.[What other descriptions should we put? - square kilometers? climate zones? topography? other?] 

Example of citation: \cite{liu_estimating_2005}

\subsection*{PM\textsubscript{2.5} Measurements}

[Write short description of each PM\textsubscript{2.5} data source.]

We downloaded the 2008-2018 pre-generated daily summary files for PM\textsubscript{2.5} (88101 and 88502 parameter codes)  (\url{https://aqs.epa.gov/aqsweb/airdata/download_files.html#Daily}) as well as the spreadsheet listing all AQS monitors with datums (\url{https://aqs.epa.gov/aqsweb/airdata/aqs_monitors.zip}) 
from the United States Environmental Protection Agency (US EPA).

All available PM\textsubscript{2.5} data in the Fire Cache Smoke Monitor Archive (\url{https://wrcc.dri.edu/cgi-bin/smoke.pl}) was downloaded for the years 2008-2018. 

PM\textsubscript{2.5} data from the Uintah Basin, Utah were provided by Seth Lyman at Utah State University (personal communication).

PM\textsubscript{2.5} data from the Persistent Cold Air Pool Study (PCAPS) \cite{Silcox_wintertime_2012} conducted in the Salt Lake Valley, Utah in January--February, 2011 were provided by Dr. Geoff Silcox in Chemical Engineering at the University of Utah


\subsection*{Predictors}

[Write short description of each predictor data set and refer to Table 1]

\subsection*{Machine learning modelling and mapping}

[Write description of ML modelling approach]

\subsection*{Code availability}

%TEMPLATE TEXT: For all studies using custom code in the generation or processing of datasets, a statement must be included in the Methods section, under the subheading "Code availability", indicating whether and how the code can be accessed, including any restrictions to access. This section should also include information on the versions of any software used, if relevant, and any specific variables or parameters used to generate, test, or process the current dataset. 

[Insert brief description of how to access code on GitHub.] The code was written and annotated in R [version number] and Python [version number] and is available from GitHub [doi citation link]. The key package for implementing the ML model was [caretEnsemble?]. 

\section*{Data Records}

%TEMPLATE TEXT: The Data Records section should be used to explain each data record associated with this work, including the repository where this information is stored, and to provide an overview of the data files and their formats. Each external data record should be cited numerically in the text of this section, for example \cite{cite3, cite4, cite5, cite6}, and included in the main reference list as described below.. A data citation should also be placed in the subsection of the Methods containing the data-collection or analytical procedure(s) used to derive the corresponding record.

%TEMPLATE TEXT: Tables should be used to support the data records, and should clearly indicate the samples and subjects (study inputs), their provenance, and the experimental manipulations performed on each (please see Tables and Submitting Experimental Metadata, below). They should also specify the data output resulting from each data-collection or analytical step, should these form part of the archived record.

All data are freely available from [repository name, data doi citation]. We provide ... [reference Figure 2]

[insert Figure 2: choropleths at zip code level - 4-panel: a) highest year PM\textsubscript{2.5}, Aug or Sept, b) highest year PM\textsubscript{2.5}, Jan/Feb, c) lowest year PM\textsubscript{2.5}, Aug or Sept, d) lowest year PM\textsubscript{2.5}, Jan/Feb.]

[Insert Table 3: list of files]

\section*{Technical Validation}

%TEMPLATE TEXT: This section presents any experiments or analyses that are needed to support the technical quality of the dataset. This section may be supported by up figures and tables, as needed. This is a required section; authors must present information justifying the reliability of their data.

[Write description of goodness of fit methods/metrics - out-of-bag data, RMSE, R2, models run on subsets of data, etc.]

[Insert Figure 4: a) out-of bag observed PM\textsubscript{2.5} vs predicted, b) full model observed PM\textsubscript{2.5} vs predicted, c-j) various subsets of data - oob and full model plots (see figure 5 of example paper)]

[Write discussion about variable importance, possibly referring to the suggested figure of variable importance panel figure. Could make an observation or two about the complexity of the variables, e.g., PM\textsubscript{2.5} can be highest at highest and lowest temperatures (summer fire season and winter inversions), etc.]

[Thoughts - insert figure of predicted PM\textsubscript{2.5} vs predictor variable for the 8 (or so) most important variables (panel figure)]

Thoughts: compare to PM\textsubscript{2.5}. Concerned comparing to HMS will take too long? 


\section*{Usage Notes}

%TEMPLATE TEXT: The Usage Notes should contain brief instructions to assist other researchers with reuse of the data. This may include discussion of software packages that are suitable for analysing the assay data files, suggested downstream processing steps (e.g. normalization, etc.), or tips for integrating or comparing the data records with other datasets. Authors are encouraged to provide code, programs or data-processing workflows if they may help others understand or use the data. Please see our code availability policy for advice on supplying custom code alongside Data Descriptor manuscripts.

%TEMPLATE TEXT: For studies involving privacy or safety controls on public access to the data, this section should describe in detail these controls, including how authors can apply to access the data, what criteria will be used to determine who may access the data, and any limitations on data use. 

[Write brief description of things the provided code can be adapted to do, such as making plots of specific years, use in health/pollution studies.]

\section*{Acknowledgements}

%TEMPLATE TEXT: The Acknowledgements should contain text acknowledging non-author contributors. Acknowledgements should be brief, and should not include thanks to anonymous referees and editors or effusive comments. Grant or contribution numbers may be acknowledged.

[Write acknowledgements text here.]

\section*{Author contributions}

%TEMPLATE TEXT: Each author’s contribution to the work should be described briefly, on a separate line, in the Author Contributions section. 

[Write brief description of contribution from each author.]

\section*{Competing interests}

%TEMPLATE TEXT: A competing interests statement is required for all papers accepted by and published in \emph{Scientific Data}. If there is no conflict of interest, a statement declaring this must still be included in the manuscript.

The authors declare not competing interests.

\section*{Figures and figures legends}

%TEMPLATE TEXT: Figure should be referred to using a consistent numbering scheme through the entire Data Descriptor. For initial submissions, authors may choose to supply this document as a single PDF with embedded figures, but separate figure image files must be provided for revisions and accepted manuscripts. In most cases, a Data Descriptor should not contain more than three figures, but more may be allowed when needed. We discourage the inclusion of figures in the Supplementary Information \textendash{} all key figures should be included here in the main Figure section. 

%TEMPLATE TEXT: Figure legends begin with a brief title sentence for the whole figure and continue with a short description of what is shown in each panel, as well as explaining any symbols used. Legend must total no more than 350 words, and may contain literature references. 

[All figures go here and are referred to in the text]

\section*{Tables}

%TEMPLATE TEXT: Authors are encouraged to provide one or more tables that provide basic information on the main ‘inputs’ to the study (e.g. samples, participants, or information sources) and the main data outputs of the study; also see the additional information on providing metadata on page 6. Tables in the manuscript should generally not be used to present primary data (i.e. measurements). Tables containing primary data should be submitted to an appropriate data repository.

%TEMPLATE TEXT: Tables may be provided within the \LaTeX{} document or as separate files (tab-delimited text or Excel files). Legends, where needed, should be included here. Generally, a Data Descriptor should have fewer than ten Tables, but more may be allowed when needed. Tables may be of any size, but only Tables which fit onto a single printed page will be included in the PDF version of the article (up to a maximum of three). 

%TEMPLATE TEXT: Due to typesetting constraints, tables that do not fit onto a single A4 page cannot be included in the PDF version of the article and will be made available in the online version only. Any such tables must be labelled in the text as ‘Online-only’ tables and numbered separately from the main table list e.g. ‘Table 1, Table 2, Online-only Table 1’ etc.

[All tables go here and are referred to in the text - read template text for tables]

%%%% top of EXAMPLE BIBLIOGRAPHY - KEEP %%%%
% INSTRUCTIONS: References should be included within the manuscript file itself as our system cannot accept BibTeX bibliography files. Authors who wish to use BibTeX to prepare their references should therefore copy the reference list from the .bbl file that BibTeX generates and paste it into the main manuscript .tex file (and delete the associated \textbackslash{}bibliography and \textbackslash{}bibliographystyle commands).
%\begin{thebibliography}{1}
%\expandafter\ifx\csname url\endcsname\relax
%  \def\url#1{\texttt{#1}}\fi
%\expandafter\ifx\csname urlprefix\endcsname\relax\def\urlprefix{URL }\fi
%\providecommand{\bibinfo}[2]{#2}
%\providecommand{\eprint}[2][]{\url{#2}}
%
%\bibitem{cite1}
%\bibinfo{author}{Califano, A.}, \bibinfo{author}{Butte, A.~J.},
%  \bibinfo{author}{Friend, S.}, \bibinfo{author}{Ideker, T.} \&
%  \bibinfo{author}{Schadt, E.}
%\newblock \bibinfo{title}{{Leveraging models of cell regulation and GWAS data
%  in integrative network-based association studies}}.
%\newblock \emph{\bibinfo{journal}{Nat. Genet.}}
%  \textbf{\bibinfo{volume}{44}}, \bibinfo{pages}{841--847}
%  (\bibinfo{year}{2012}).
%
%\bibitem{cite2}
%\bibinfo{author}{Wang, R.} \emph{et~al.}
%\newblock \bibinfo{title}{{PRIDE Inspector: a tool to visualize and validate MS
%  proteomics data.}}
%\newblock \emph{\bibinfo{journal}{Nat. Biotechnol}}
%  \textbf{\bibinfo{volume}{30}}, \bibinfo{pages}{135--137}
%  (\bibinfo{year}{2012}).
%
%\bibitem{cite3}
%Zhang, Q-L., Chen, J-Y., Lin, L-B., Wang, F., Guo, J., Deng, X-Y. Characterization of ladybird Henosepilachna vigintioctopunctata transcriptomes across various life stages. \emph{figshare} https://doi.org/10.6084/m9.figshare.c.4064768.v3 (2018).
%
%\bibitem{cite4}
%\emph{NCBI Sequence Read Archive} http://identifiers.org/ncbi/insdc.sra:SRP121625 (2017).
%
%\bibitem{cite5}
%Barbosa, P., Usie, A. and Ramos, A. M. Quercus suber isolate HL8, whole genome shotgun sequencing project. \emph{GenBank} http://identifiers.org/ncbi/insdc:PKMF00000000 (2018).
%
%\bibitem{cite6}
%\emph{DNA Data Bank of Japan} http://trace.ddbj.nig.ac.jp/DRASearch/submission?acc=DRA004814 (2016).
%
%\end{thebibliography}
%%%% bottom of EXAMPLE BIBLIOGRAPHY - KEEP %%%%

%%%% Bibliography - copy .bbl file into example above and delete next two lines %%%%
\bibliographystyle{plain}
\bibliography{../ReidGroupReferences}

%\subsection*{Citing Data}
%In line with emerging industry-wide standards for data citation, references to all datasets described or used in the manuscript should be cited in the text with a superscript number and listed in the ‘References’ section in the same manner as a conventional literature reference. See the examples above.

\end{document}
